eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-shelley-test
-version:            1.3.0.2
+version:            1.3.1.0
```

There was no change to the package, why version bump?
```suggestion
version:            1.3.0.2
```